yarn.lock

```diff
@@ -4676,6 +4676,11 @@
     "@types/scheduler" "*"
     csstype "^3.0.2"
 
+"@types/reactjs-localstorage@^1.0.0":
```

Comment:
```suggestion
"@types/reactjs-localstorage@1.0.0":
```
Run `yarn` again and lockfile should be updated with this new dependency version